apps/src/applab/DesignToolbox.jsx

```diff
@@ -100,6 +100,12 @@ export default class DesignToolbox extends React.Component {
           elementType={'SLIDER'}
           handleDragStart={this.props.handleDragStart}
         />
+        <DesignToolboxElement
+          imageUrl={IMAGE_BASE_URL + 'camera.png'}
+          desc={'Photo Chooser'}
+          elementType={'PHOTO_CHOOSER'}
+          handleDragStart={this.props.handleDragStart}
+        />
```

Comment:
Two quick thoughts on the UI for this element. 
1. the camera looks a lot bigger than the other design elements. Suggestion: decrease the size of camera.png or add a style attribute that sets the width. Width of 50-55px seemed more normal when I was playing around with the HTML.
<img src="https://user-images.githubusercontent.com/8324574/81615461-7d03cd00-9396-11ea-8bfc-58392d9b02dc.png" width="150"/>